### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by amirnathani9**
# change logs
-Added Vertical Card
-Added Horizontal Card
-Card with Badge
-Card with Text Overlay
-Card with Shadow
-Only Text Card

## Gif

card new

## Preview Link

https://deploy-preview-3--nomark-ui.netlify.app/components/card/card

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
+.card{
+    display: flex;
+    border: 1px solid var(--primary-color);
+}
+.card-vertical{
+    flex-direction: column;
+    position: relative;
+    max-width: 302px;
```
Comment (devdeadviz) [inline components/card/card.css]: Avoid using absolute units, until it's much required.

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    color: var(--primary-color);
+}
+
+.card-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.card-content-horizontal{
+    margin: 5px 20px;
```
Comment (devdeadviz) [inline components/card/card.css:46]: same here

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+}
+.card-text-overlay>span{
+    color: var(--text-color-1) ;
+    font-size: 3rem;
+    margin: auto;
+    background-color: rgba(2, 129, 149, 0.5);
+}
+
+.card-shadow{
+    box-shadow: 10px 10px 50px rgb(0, 0, 0, 0.8);
```
Comment (devdeadviz) [inline components/card/card.css]: Try to use color variables instead of hard-coding them.

**Review by devdeadviz — COMMENTED** [review]
Added some suggestions have a look at it.

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    position: absolute;
+    z-index: 1;
+    background-color: rgba(0, 0, 0, 0.8);
+}
+.card-text-overlay>span{
+    color: var(--text-color-1) ;
+    font-size: 3rem;
+    margin: auto;
+    background-color: rgba(2, 129, 149, 0.5);
+}
```
Comment (bishnoimukesh) [inline components/card/card.css]: either use color variable or rgb

File: styles/snippet.css [hunk 5]
```diff
@@ -1,12 +1,11 @@
 
 pre{
-    white-space: pre-line;
     max-height: 350px;
     overflow-y: auto;
 }
 code{
+    padding: 0px !important;
     line-height: 2;
-    padding: 2rem !important;
```
Comment (bishnoimukesh) [inline styles/snippet.css:9]: avoid using important

**Review by bishnoimukesh — APPROVED** [review]
few changes are suggested for consistency. Rest all good to merge.

File: styles/snippet.css (hunk 5 above)
Comment (amirnathani9) [reply styles/snippet.css:9]: here the pre tag gives auto padding to overwrite that I have to give important.

File: components/card/card.css (hunk 1 above)
Comment (amirnathani9) [reply components/card/card.css]: changed

File: components/card/card.css (hunk 2 above)
Comment (amirnathani9) [reply components/card/card.css:46]: its smaller value thats why using it

Comment (amirnathani9) [reply components/card/card.css:44]: its smaller value thats why using it

File: components/card/card.css (hunk 3 above)
Comment (amirnathani9) [reply components/card/card.css]: changes done

File: components/card/card.css (hunk 4 above)
Comment (amirnathani9) [reply components/card/card.css]: done the changes

**Pull request merged**